ha/core/system_health/system_health.py

```diff
@@ -442,6 +442,7 @@ def process_event(self, healthevent: HealthEvent):
                     # If health is already stored and its a node_health, check further
                     stored_genration_id = current_health_dict["events"][0]["specific_info"]["generation_id"]
                     stored_status = current_health_dict["events"][0]["status"]
+                    stored_functional_type = current_health_dict["events"][0]["specific_info"]["functional_type"]
```

Comment:
functional_type was populated only in some of the cases in parse_event pf parser.py
Will it be available here in all scenarios ?